tests/openapi.test.ts

```diff
@@ -6,7 +6,7 @@ import { generate } from '../src'
 test('Should generate the right definition', async (t) => {
   await generate({
     tsconfigFilePath: path.resolve(__dirname, './fixture/tsconfig.json'),
-    controllers: [path.resolve(__dirname, './fixture/con*.ts')],
+    controllers: [path.resolve(__dirname, './fixture/controller.ts')],
```

Comment:
i believe the test was also meant the test the matching of files based on regex, could we do this intead ? 
```suggestion
    controllers: [path.resolve(__dirname, './fixture/controller.*')],
```

Reply:
Just a note, this can be a bit tricky to spot, when a new controller test is added, all controllers get merged into `/tmp/openapi-test-valid.json`, which might not match the snapshot in `./fixture/openapi.json`. I've been thinking about restructuring the tests a bit and switching to the native `node:test` runner with assertions and snapshots to make this more robust.

That said, it's outside the scope of this PR.